src/main/resources/templates/home.html

```diff
@@ -0,0 +1,33 @@
+<html>
+<body>
+<a href="/">All</a>
+<a href="/?category=Furniture">Furniture</a>
+<a href="/?category=Alcohol">Alcohol</a>
+<a href="/?category=Toiletries">Toiletries</a>
+<a href="/?category=Shoes">Shoes</a>
+<a href="/?category=Food">Food</a>
+<a href="/?category=Jewelry">Jewelry</a>
+<form action="/" method="get">
+    <input type="text" placeholder="Enter name" name="name">
+    <button type="submit">Submit</button>
+</form>
+
+<form action="/" method="get">
+    <input type="text" placeholder="Search" name="search"/>
+    <button type="submit">Search</button>
+</form>
+<ol>
+    {{#purchases}}
+    <li>{{date}} {{creditCard}} {{cvv}} {{category}}</li>
+    {{/purchases}}
+
+</ol>
+<br>
```

Comment:
Purchases and customers should be displayed in the same table.  With the name and email of the customer that made the purchase next to what they purchased.